hedera-node/hedera-app/src/xtest/java/contract/Erc721OperationsTest.java

```diff
@@ -183,12 +124,11 @@ void setUp() {
     void erc721OperationsResultInExpectedState() throws IOException {
```

Comment:
Will this single test exercise all of the operations?  If not maybe it should be renamed to something like `erc721ContractCreateOperationResultInExpectedState` which is awfully long name.

Reply:
Renamed to `recordedErc721OperationsResultInExpectedState()` to hopefully clarify scope a bit.